packages/frontend/src/components/ActionPanel/actionPanel.module.css

```diff
@@ -104,3 +108,20 @@
     }
   }
 }
+
+@media (max-width: 600px /* todo: what is mobile breakpoint */) {
```

Comment:
Vi burde kanskje lande det en gang for alle.
irriterende at man ikke kan bruke css-variabler i media queries, det hadde vært nyttig

Reply:
Enig. Kanskje vi på sikt kan lage oss en linte regel for det?